shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -262,6 +262,15 @@ object GlutenConfig {
   val GLUTEN_CLICKHOUSE_SEP_SCAN_RDD_DEFAULT = true
   val GLUTEN_CLICKHOUSE_CONFIG_PREFIX = "spark.gluten.sql.columnar.backend.ch"
 
+  // For Soft Affinity Scheduling
+  val GLUTEN_SOFT_AFFINITY_ENABLED = "spark.gluten.soft-affinity.enabled"
+  val GLUTEN_SOFT_AFFINITY_ENABLED_DEFAULT_VALUE = false
+  val GLUTEN_SOFT_AFFINITY_REPLICATIONS_NUM = "spark.gluten.soft-affinity.replications.num"
```

Comment:
Can you please help to write a short note on how to use this? does this mean by providing two "fake" replicas, then spark will schedule to the first place?

Reply:
add the doc for these configs, please take a review